Program.cs

```diff
@@ -3,17 +3,52 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to sort and swap special characters.
+// Given a char array, a special character S and sort order (default order Ascending), print sorted array keeping elements matching S at the last.
 // ---------------------------------------------------------------------------------------
-using static System.Console;
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Sort and Swap Array</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method gets valid user input and displays the sorted and swapped array</summary>
+      static void Main () {
+         char[] letters;
+         for (int i = 0; ; i++) {
+            Console.Write ("Enter the characters to be sorted (as string): ");
+            letters = Console.ReadLine ().ToLower ().ToCharArray ();
+            if (letters.Any (x => !char.IsLetter (x))) {
+               Console.Write ("Only alphabets are allowed.\n");
+               continue;
+            } else break;
+         }
+         string splCharInput;
+         for (int i = 0; ; i++) {
+            Console.Write ("Enter the special character to be swapped to the end: ");
+            splCharInput = Console.ReadLine ();
+            if (splCharInput.Length != 1 || splCharInput.Any (x => !char.IsLetter (x))) {
+               Console.Write ("Invalid Input.\n");
+               continue;
+            } else break;
+         }
+         char splChar = splCharInput.ToCharArray ()[0];

```

Comment:
Since its only one character here no need to convert this to a char array. For a string, each character can be accessed using the indexer. So, this can simply be:
_char splChar = splCharInput[0];_